avocado/core/remoter.py

```diff
@@ -173,3 +175,12 @@ def receive_files(self, local_path, remote_path):
         except ValueError:
             return False
         return True
+
+
+class FabricException(Exception):
+
+    def __init__(self, msg):
```

Comment:
also I'm not sure if that's a strong requirement, but I'm used to look for Exceptions on top of the file, what do you think?